### PR TITLE
feat: 添加 2.0 fileMin 压缩插件

### PR DESCRIPTION
**Pull request by itboos**
#2334

### CODE REVIEW

File: packages/wepy-plugin-filemin/package.json [hunk 1]
```diff
@@ -1,17 +1,29 @@
 {
-  "name": "wepy-plugin-filemin",
-  "version": "1.3.12",
-  "description": "wepy filemin plugin",
+  "name": "sm-wepy-plugin-file-min",
```
Comment (dlhandsome) [inline packages/wepy-plugin-filemin/package.json]: 官方插件需以 ```@wepy/plugin-xxx``` 形式命名

File: packages/wepy-plugin-filemin/package.json [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+  "homepage": "https://github.com/itboos/sm-wepy-plugin-file-min",
```
Comment (dlhandsome) [inline packages/wepy-plugin-filemin/package.json]: 这里也需要调整下

File: packages/wepy-plugin-filemin/src/index.js [hunk 3]
```diff
@@ -1,46 +1,36 @@
...
-            });
-
-            if (/\.(wxml|xml)$/.test(op.file)) {
-                op.code = pd.xmlmin(op.code);
-            } else if (/\.json$/.test(op.file)) {
-                op.code = pd.jsonmin(op.code);
-            }
-            op.next();
+    config = Object.assign(config, options)
+    this.register('build-components', comps => {
```
Comment (dlhandsome) [inline packages/wepy-plugin-filemin/src/index.js]: ```build-app```、```build-vendor``` 是否也需要考虑到呢

File: packages/wepy-plugin-filemin/LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (Gcaufy) [inline packages/wepy-plugin-filemin/LICENSE]: 插件 repo 不需要单独添加 license 哈，以主 repo license 为主。

File: packages/wepy-plugin-filemin/.gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+package-lock.json
```
Comment (Gcaufy) [inline packages/wepy-plugin-filemin/.gitignore]: .gitignore 在主 repo 中也是存在的

File: packages/wepy-plugin-filemin/src/index.js [hunk 6]
```diff
@@ -1,46 +1,36 @@
-/**
```
Comment (Gcaufy) [inline packages/wepy-plugin-filemin/src/index.js]: 这里还需要参照其它 plugin, 添加一下 test case

**Comment by Gcaufy** [comment]
感谢 PR~~ 评论已添加。

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2360) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2360) before we can accept your contribution.<br/><hr/>**zhongdonglin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/wepy?pullRequest=2360) it.</sub>

**Pull request closed without merging**